.github/workflows/build.yml

```diff
@@ -134,10 +112,6 @@ jobs:
           path: node_modules
           key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-yarn
 
-      - name: Building NodeJS dependencies - host OS
```

Comment:
probably ditto?

Reply:
actually we put both dependence updates in the same script